### ISSUE TITLE
Hide or Disable Adult Categories 

### ISSUE THREAD

**Issue by tozfek** [Prowlarr/Prowlarr#1859]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

No, it would be nice to have the ability to hide all Adult Categories from prowlarr. 

### Describe the solution you'd like

Ability to hide all Adult Categories from prowlarr. 

### Describe alternatives you've considered

I realize that searches can be limited by selecting categories to search against but they are occasionally updated or and from time to time obscure matches are made after a search. 

### Anything else?

As a recovering porn addict just seeing the xxx is triggering. I have used adblock to hide the adult categories and xxx tag from the list of indexers. I have disabled them on the indexers themselves but I realize that is not how the searches are conducted.

**Comment by bakerboy448** [comment]
> I realize that searches can be limited by selecting categories to search against but they are occasionally updated or and from time to time obscure matches are made after a search.

if adult categories are not selected for a search then there wouldn't be any results displayed for them.  

So is this request to have a toggle to show/hide them as selectable in the UI where category selection can occur? Search, Application Settings, and Download Client Settings?  

otherwise not getting adult category results is as simple as not doing adult category searches 🫠

**Comment by thezak48** [comment]
Do you mean if you do a search with no category selected you want xxx categories to be hidden from the results?

**Comment by tozfek** [comment]
> Do you mean if you do a search with no category selected you want xxx categories to be hidden from the results?

Yes, such that when no category is selected and a search is performed no adult results are displayed.

**Comment by NaPurHab** [comment]
I think I understand the request. What @tozfek is requesting is to have an entry/switch in Settings that hides the XXX Adult Categories from the Categories Selector. Much like going to a video store, and seeing a Black Curtain over a doorway. Making it child friendly. Or as he said:

> As a recovering porn addict just seeing the xxx is triggering.

So simply add a switch in settings to hide the XXX Categories. Then it would not show in the main interface.

Of course, it does nothing for material that is not properly categorised.

**Comment by tozfek** [comment]
Yes, exactly! Thank you for understanding the request. 

> So simply add a switch in settings to hide the XXX Categories. Then it would not show in the main interface.

To just expand on the above, it would be nice to also remove the adult categories from the search if enabled. 

> Of course, it does nothing for material that is not properly categorized.

agreed

**Comment by tozfek** [comment]
Not to rush, @bakerboy448 but is there an ETA on when this would be prioritized?

**Comment by matheusroberson** [comment]
I was looking for something like this—making Prowlarr more family-friendly by allowing it to exclude XXX categories. I can implement the necessary filters on the frontend, but I'm not sure how to add a new configuration parameter in the database/backend. If someone help me with this part, i can make a PR for this
